src/models/Block.ts

```diff
@@ -0,0 +1,14 @@
+export interface IBlock {
+  _id: string;
+  height: number;
+  status: string;
+  anchor: number;
```

Comment:
```suggestion
  anchor: string;
```

Reply:
But sanctuary returns a number, do i need to convert it manually in sdk?

Reply:
anchor is BigNumber because its number from blockchain, so if sanctuary returns number we need to fix that, please crate task in Jira or pleas open PR for sanctuary directly if this is quick fix

Reply:
Fixed it in sanctuary and here in sdk